activerecord/lib/active_record/runtime_registry.rb

```diff
@@ -10,13 +10,6 @@ module ActiveRecord
   # See the documentation of ActiveSupport::PerThreadRegistry
   # for further details.
   class RuntimeRegistry # :nodoc:
-    extend ActiveSupport::PerThreadRegistry
-
-    attr_accessor :connection_handler, :sql_runtime, :connection_id
-
-    [:connection_handler, :sql_runtime, :connection_id].each do |val|
-      class_eval %{ def self.#{val}; instance.#{val}; end }, __FILE__, __LINE__
-      class_eval %{ def self.#{val}=(x); instance.#{val}=x; end }, __FILE__, __LINE__
-    end
```

Comment:
This place is one of the places that it doesn't make sense. Before we had one thread local variable. Now we have three. I prefer to keep the current implementation. Thanks